### PR TITLE
dev-python/netifaces: Fix incompatible pointer to integer conversion

### PR DESCRIPTION
**Pull request by listout**
Closes: https://bugs.gentoo.org/894448

### CODE REVIEW

**Comment by listout** [comment]
Upstream seems to have halted development. https://github.com/al45tair/netifaces

File: dev-python/netifaces/files/netifaces-0.11.0-musl-clang16-null.patch [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+ 
+   if (bigaddr) {
+@@ -1805,7 +1809,11 @@ gateways (PyObject *self)
+         sizeof(sanl_from),
+         &iov,
+         1,
++#ifdef __GLIBC__
+         NULL,
++#else
++		0,
```
Comment (thesamesam) [inline dev-python/netifaces/files/netifaces-0.11.0-musl-clang16-null.patch:23]: Indentation is wrong.

File: dev-python/netifaces/files/netifaces-0.11.0-musl-clang16-null.patch [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+--- a/netifaces.c
++++ b/netifaces.c
+@@ -370,7 +370,11 @@ string_from_sockaddr (struct sockaddr *addr,
+ 
+   failure = getnameinfo (gniaddr, gnilen,
+                          buffer, buflen,
++#ifdef __GLIBC__
+                          NULL, 0,
++#else
++                         0, 0,
```
Comment (thesamesam) [inline dev-python/netifaces/files/netifaces-0.11.0-musl-clang16-null.patch:11]: Can you unconditionally do 0?

Comment (listout) [reply dev-python/netifaces/files/netifaces-0.11.0-musl-clang16-null.patch:11]: Sure.

**Comment by listout** [comment]
Uhh ... you said unconditionally :/

**Review by mgorny — APPROVED** [review]
Thanks!

**Pull request closed without merging**